tools/release_tool.py

```diff
@@ -21,10 +24,24 @@ def run_command(description, args, capture_output=True, shell=True):
     printed_args = args.join(" ") if type(args) == list else args
     log(printed_args, command=True)
     stdout = subprocess.PIPE if capture_output else None
-    completed_process = subprocess.run(args, stdout=stdout, shell=shell, check=True, encoding="utf-8")
+    completed_process = subprocess.run(args, stdout=stdout, stderr=stdout, shell=shell, check=True, encoding="utf-8")
```

Comment:
Configuring `stderr` this way causes us to lose the compiler output on the terminal. (For comparison, run `python release_tool.py preflight` with and without this change.) Is it necessary to do that? The compiler output is helpful, so it'd be good to retain it if possible.

Reply:
I debated that as well. It made sense to me given that it's named capture_output to capture both stdout and stderr. The only reason I made the change was because repeating the failed output to the console seemed a bit much. I'm fine with either.

Reply:
I'll also note, I didn't find semvar's errors particularly useful since, in many cases though not this one specifically since pip writes to stderr afaik, it only returns if it failed, and not what the error was. Capturing the stderr and returning it allows the code to handle it better in cases where that isn't the case.